src/Button/index.scss

```diff
@@ -38,12 +38,6 @@
 }
 
 /* Kind variants */
-.nds-button--primary,
-.nds-button--secondary,
-.nds-button--tonal {
-  @extend %btn-kind-base;
-}
```

Comment:
No longer needed - the `%btn-kind-base` placeholder was removed in this PR because those styles are coupled to the `size` prop.